tests/testutils/tracee.go

```diff
@@ -16,13 +16,13 @@ import (
 const (
 	readinessPollTime           = 200 * time.Millisecond
 	httpRequestTimeout          = 1 * time.Second
-	TraceeDefaultStartupTimeout = 10 * time.Second
+	TraceeDefaultStartupTimeout = 15 * time.Second
 )
 
 var (
 	TraceeBinary   = "../../dist/tracee"
 	TraceeHostname = "localhost"
-	TraceePort     = 3366
+	TraceePort     = 3369
```

Comment:
The ideia is also change all referente to port 3366 ? Like in the Vagrantfile?

Reply:
No, it's just for these integration tests, since they are probably in conflict with Vagrant machines.